docker/run-supervisor.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+
+set -xe
+
+enable_proc() {
+  cp "etc/supervisor_available/$1.conf" "etc/supervisor_enabled/$1.conf"
```

Comment:
I tried using a symlink instead and supervisor didn't seem to like it :disappointed: